app/src/main/java/com/lab/skuld/ui/screens/SettingsScreen.kt

```diff
@@ -129,7 +130,7 @@ fun ImportButton() {
         val tasksJson = file.readText()
         val tasks = gson.fromJson(tasksJson, Array<Task>::class.java)
         val eventNoIDs = tasks.map { task ->
-            EventNoID(
+            CreateEvent(
```

Comment:
it won't export dates, so can we consider that as a future-proofing feature or is that my problem from now on?